new-pipeline/dist.html

```diff
@@ -47,8 +47,6 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                     <select id="filter-os" class="multiselect" title="Target OS" data-all-selected="any OS" data-n-selected="OSs" multiple></select>
                     <select id="filter-arch" class="multiselect" title="Product architecture" data-all-selected="any architecture" data-n-selected="architectures" multiple></select>
                     with
-                    <select id="filter-e10s" class="multiselect" title="e10s setting" data-all-selected="any e10s setting" multiple></select>
-                    as
```

Comment:
After removing this, I would expect that the javascript will start throwing errors when it can no longer find the multiselect. Are there errors in the log?